internal/telemetry/service_test.go

```diff
@@ -9,8 +9,7 @@ import (
 
 func TestNewService(t *testing.T) {
 	t.Run("Should create the expected Service", func(t *testing.T) {
-		service := NewService(ModeOn, "userID", "command")
-
```

Comment:
keep the newline here, in general I'd suggest trying to keep the amount of diffs only to what's necessary.  the newline stuff (adding/removing) just adds noise to the PRs